en/admin/installation/installation.md

```diff
@@ -9,6 +9,12 @@ git clone https://github.com/wallabag/wallabag.git
 cd wallabag && make install
 ```
 
+If you want to use LDAP, replace last command with
+```bash
+cd wallabag && make install LDAP_ENABLED=true
+```
```

Comment:
Because we want to have an installation setup as easy as possible, for me it's a problem, yes. Don't forget that we have users who don't know anything to web server configuration, script installation and LDAP.
But maybe I'm wrong.